en/micro-integrator/docs/use-cases/examples/template_examples/using-endpoint-templates.md

```diff
@@ -28,50 +28,117 @@ For example, let's say we have two default endpoints with following hypothetical
       <retriesBeforeSuspension>3</retriesBeforeSuspension>
       <retryDelay>0</retryDelay>
     </markForSuspension>
-  </default>
+  </address>
 </endpoint>
 ```
 
 We can see that these two endpoints have different set of error codes and different progression factors for suspension. Furthermore, the number of retries is different between them. By defining an endpoint template, these two endpoints can be converged to a generalized form. This is illustrated in the following:
```

Comment:
Do not use "we". Suggestion: "Note that these two endpoints have......"

Reply:
Done